### ISSUE TITLE
Documentation still needs to be done for fetch_survey() date-time feature update in #263

### ISSUE THREAD

**Issue by jmobrien** [ropensci/qualtRics#265]
Neglected to do this, will take care of soon.

**Comment by jmobrien** [comment]
Commit 524d9c8 has this, but it was folded into PR #266 that's currently held up while Qualtrics fixes the problem with their update survey metadata endpoint.  Should we pull it out and add it separately?

**Comment by juliasilge** [comment]
That might be smart, so the docs are not held up by things out of our control. 👍
